src/towncrier/check.py

```diff
@@ -55,6 +64,10 @@ def __main(comparewith, directory, config):
         click.echo("On trunk, or no diffs, so no newsfragment required.")
         sys.exit(0)
 
+    if files_changed == config["filename"]:
+        click.echo("Only the configured news file has changed.")
```

Comment:
I would say add uppercase when skipped, to help detect accidental configuration errors.

```suggestion
        click.echo("Checks SKIPPED. Only news file changes detected that looks like a release branch.")
```